isda_daycounters/thirtyE360ISDA.py

```diff
@@ -0,0 +1,40 @@
+"""
+ISDA 30E/360 (ISDA) day count convention.
+=====
+
+Implemented according to 2006 ISDA definition:
+https://www.isda.org/2008/12/22/30-360-day-count-conventions/
+"""
+
+import calendar
+
+name = 'thirtyE/360 ISDA'
+
+def day_count(start_date, end_date, is_end_date_on_termination=False):
```

Comment:
From my understanding, what is usually provided _is_ the termination date, right? Thus it might make more sense for this optional parameter to be `True` by default?

Reply:
It will be more often that accrual end date will not coincide with termination date since there are many more coupon dates vs only 1 termination date. Therefore my choice of making is_end_date_on_termination as false by default.

Another thing I considered was to have user input termination date every time and then make this determination. But that will be more onerous to input.

Reply:
Ah I was misinterpreting "termination date", that sounds sensible to me. 

And agreed, that seems like a bit much. Looks good as is then.